docs/kubernetes/flannel-bigip-info.rst

```diff
@@ -0,0 +1,143 @@
+.. _flannel-bigip-info:
+
+Flannel VXLAN, Kubernetes, and BIG-IP
+=====================================
+
+Cluster Networking in Kubernetes with Flannel
+---------------------------------------------
+
+Be sure to read about Kubernetes `Cluster Networking`_ to understand the theory behind this strategy.
+
+The basic gist of this approach is that we have a Kubernetes cluster with multiple Nodes, Kubernetes creates Pods across the Nodes, 
+and we want these Pods to be able to communicate with each other without NAT.
+In this article, we will implement this using Flannel's VXLAN configuration.
+
+`Flannel`_ is a way to configure a layer 3 network fabric. In older versions of Kubernetes (pre-1.6),
+Flannel solely used an etcd key-value store to read and write information about Kubernetes Nodes.
+This information included the VTEP (VXLAN tunnel endpoint) of each Node, as well as a subnet within the defined Flannel network range.
+Flannel assigns a different subnet to each Node, and allocates all Pod IP's within these subnets (on their respective Nodes).
+In more recent versions of Kubernetes, Flannel's preferred mode of operation is via the Kubernetes API.
+Etcd is still viable, but is not as common. Flannel provides a `kube-flannel.yml`_ file for use
+as a network addon to kubeadm. This creates a Flannel Pod on each Kubernetes node that runs two containers: install-cni and flanneld.
+Each instance of flanneld now monitors the Kubernetes API for the Nodes that live in the cluster.
+The Nodes will have flannel-specific annotations attached to them, containing similar information to what etcd stores.
+
+Here are the supported annotations:
```

Comment:
@sjberman - This bit about the Annotations is a little confusing. They come from json blobs included in the data section of the ConfigMap, correct? Do they appear as Annotations when you get the Node description?

Reply:
These aren't involved in any ConfigMap. They are simply annotations that are configured as part of the Node description, yes. Does that answer your question?